echopype/echodata/combine.py

```diff
@@ -124,6 +165,10 @@ def combine_echodata(echodatas: List[EchoData], combine_attrs="override") -> Ech
     old_time2 = None
     # mru time after reversal correction
     new_time2 = None
+    # time3 before reversal correction
+    old_time3 = None
+    # time3 after reversal correction
+    new_time3 = None
```

Comment:
Why do all these `old_*` and `new_*` variables need to be declared? I see that in `check_and_correct_reversed_time`, `old_time` is tested for `None`. But as far as I can see, the `old_time` value passed to that function will always be `None`, at this time. Same for `new_time`: the value passed to `check_and_correct_reversed_time` seems like it's always `None` at this time. The function arguments `old_time` and `new_time` could be changed from positional to keyword arguments with default `None` values; or removed altogether, but then you'd have tweak the function code itself.

Reply:
From my understanding the code right now basically uses `old_*` and `new_*` as flags. You are right that how it is done does not make too much sense and we could probably create a better solution to this. I suggest that we open up a new issue and deal with this after 0.6.0, @emiliom if you agree, can you open up an issue for this?

Reply:
Thanks. That makes sense, and I'll open the issue.